common/rate-limiter/integration-test/src/main.rs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use futures::executor::block_on;
+
+fn main() {
```

Comment:
oh, I see you went with a main function, then let's just embed that in as the default main for rate limiter or rename it using something like this: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries